<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlobDetectorWidget</class>
 <widget class="QWidget" name="BlobDetectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="numberOfAgentsLabel">
       <property name="text">
        <string>Number of agents</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QSpinBox" name="numberOfAgentsEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="value">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveSettingsButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../cats/icons.qrc">
           <normaloff>:/resources/icons/ic_save_black_24dp.png</normaloff>:/resources/icons/ic_save_black_24dp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="minBlobSizeLabel">
       <property name="text">
        <string>Minimal blob size</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="minBlobSizeSpinBox">
       <property name="suffix">
        <string>[px]</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="qualityLevelLabel">
       <property name="text">
        <string>Quality level</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="qualityLevelEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="minDistanceLabel">
       <property name="text">
        <string>Minimal distance</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="minDistanceSpinBox">
       <property name="suffix">
        <string>[px]</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="blockSizeLabel">
       <property name="text">
        <string>Block size</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="blockSizeSpinBox">
       <property name="suffix">
        <string>[px]</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="kLabel">
       <property name="text">
        <string>k value</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="kEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="useHarrisDetectorCheckBox">
     <property name="text">
      <string>Use Harris detector</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../cats/icons.qrc"/>
 </resources>
 <connections/>
</ui>
